### PR TITLE
Enhancing File Control

### PR DESCRIPTION
**Pull request by infinitytmbots**
**Please describe the changes this PR makes and why it should be merged:**
The old pull request had to be closed due an error popped by the repository. Added the visibility factor.
**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [x] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/file_control_MOD.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ class FileControl {
     this.name = 'File Control'
     this.section = 'File Stuff'
     this.fields = ['input', 'format', 'filename', 'filepath', 'filetask', 'input2']
+    this.fields = ['input', 'format', 'filename', 'filepath', 'filetask', 'input2', 'togglestatus']
```
Comment (almeidx) [inline actions/file_control_MOD.js:6]: why didn't you just add it to the line above?

File: actions/file_control_MOD.js [hunk 2]
```diff
@@ -216,8 +293,6 @@ class FileControl {
     }
     this.callNextAction(cache)
   }
-
   mod () {}
 }
-
```
Comment (almeidx) [inline actions/file_control_MOD.js:222]: add this back

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by DetroitWhiskey136 — CHANGES_REQUESTED** [review]
fix eslint errors/warnings please

**Comment by infinitytmbots** [comment]
okay

**Review by DetroitWhiskey136 — APPROVED** [review]
lgtm

**Pull request merged**